priv/repo/migrations/20190613223618_add_permissions_to_hubs.exs

```diff
@@ -0,0 +1,10 @@
+defmodule Ret.Repo.Migrations.AddPermissionsToHubs do
+  use Bitwise
+  use Ecto.Migration
+
+  def change do
+    alter table("hubs") do
+      add(:member_permissions, :integer, default: (1 <<< 4) - 1)
```

Comment:
this one's a little magical :) perhaps a comment and/or refactor. isn't this 0x07? you can probably reference the constants from here, so maybe just write out the expression?